pythonfmu/tests/test_fmi2slave.py

```diff
@@ -15,17 +15,25 @@
 
 @pytest.mark.parametrize("model", ["theModelName", None])
 def test_Fmi2Slave_constructor(model):
+
     class Slave(Fmi2Slave):
+
         modelName = model
+
+        def __init__(self, instance_name):
+            super().__init__(instance_name)
```

Comment:
This is unneeded if you just pass the arguments to super.

Reply:
` super().__init__(instance_name)` you mean?

Reply:
Ahh, I get it now